package-lock.json

```diff
@@ -1516,1568 +1516,3910 @@
         }
       }
     },
-    "@types/babel-types": {
-      "version": "7.0.4",
-      "resolved": "https://registry.npmjs.org/@types/babel-types/-/babel-types-7.0.4.tgz",
-      "integrity": "sha512-WiZhq3SVJHFRgRYLXvpf65XnV6ipVHhnNaNvE8yCimejrGglkg38kEj0JcizqwSHxmPSjcTlig/6JouxLGEhGw=="
-    },
-    "@types/babylon": {
-      "version": "6.16.3",
-      "resolved": "https://registry.npmjs.org/@types/babylon/-/babylon-6.16.3.tgz",
-      "integrity": "sha1-wpN4E6ify155oABi/EqLFD5yN7s=",
+    "@babel/code-frame": {
+      "version": "7.10.4",
+      "resolved": "https://a0us.jfrog.io/a0us/api/npm/npm/@babel/code-frame/-/code-frame-7.10.4.tgz",
```

Comment:
@mcastany what do you want to do about the jfrog references?

Reply:
damn, we need to update them

Reply:
I'll keep it as is. We'll change it in the future